ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -29,22 +29,27 @@
 #include <wiredtiger.h>
 #include <wiredtiger_ext.h>
 
-#include <aws/core/Aws.h>
 #include "s3_connection.h"
+#include "s3_log_system.h"
+#include <aws/core/Aws.h>
+#include <aws/core/utils/memory/stl/AWSString.h>
+#include <aws/core/utils/logging/AWSLogging.h>
 
 #define UNUSED(x) (void)(x)
 
 /* S3 storage source structure. */
 typedef struct {
     WT_STORAGE_SOURCE storageSource; /* Must come first */
     WT_EXTENSION_API *wtApi;         /* Extension API */
+    uint32_t awsVerboseLevel;
 } S3_STORAGE;
 
 typedef struct {
     /* Must come first - this is the interface for the file system we are implementing. */
     WT_FILE_SYSTEM fileSystem;
-    S3_STORAGE *s3Storage;
-    S3Connection *conn;
+    S3_STORAGE *storage;
+    S3Connection *connection;
+    S3LogSystem *log;
```

Comment:
I wonder if we should have the log system part of the storage source instead of the file system. The log functions class offers look thread-safe, so potentially we could have one logging module for the whole storage source, instead of one for each file system.

Maybe this was thought of and discussed and I am missing something here. We can have this discussion outside the scope of this ticket.